cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -11,21 +11,27 @@ damping_method_enum = gen.enum([
                        gen.const("truncation",          int_t, 2, "truncation")],
                      "enum types for the damping_method")
```

Comment:
for the damping there should be the following enum types:
 - None, i.e. damping_factor = 0.0
 - Constant, i.e. constant damping_factor
 - Manipulability, i.e. damping_factor based on manipulability

Truncation is no damping method anymore